### ISSUE TITLE
Redundant and verbose Phase settings tab

### ISSUE THREAD

**Issue by Flieh** [zond/android-diplicity#34]
When  viewing historical phases (any accept the current), the user is presented with the Phase result tab which indicates which players were NMR, which were active but did not tick Ready and those who ticked Ready. This is good and valid information BUT is presented again in a different form on the Phase settings page with the additional information of who ticked the Draw box. IMHO, this Phase settings page could be done away with entirely as the pertinent info is already given  in the Phase results page AND the Draw information should be kept secret. Should not the Phase settings page only be visible during the active phase? This might also allow a bit more exposition of the meaning of each box on that page.

**Comment by zond** [comment]
When I created Droidippy I felt that the draw setting should be kept secret forever.

I felt very stabby, and wanted anonymity for the sake of anonymity and secrecy for the sake of secrecy.

I have since changed my mind - maybe because an overabundance of secrecy and anonymity has hurt the community of Droidippy - and feel that it might be reasonable to show the votes from previous phases just like you see the orders submitted previous phases.

Apart from that they are a bit redundant yes, but I kind of like the symmetry in that every phase has a certain tab.

I even feel that it would be good for completeness if even the current phase had a phase result tab and even unfinished games had game results tabs. Those wouldn't make any sense at all, of course, so I don't display them. But the phase results for old phases makes sense (if slightly redundant).

Does this make sense?

I'm not fully commited in any direction, but I need a thought through discussion before changing it :)

**Comment by LlamaPlays** [comment]
I think the draw should still be kept a secret as like you've mentioned, makes it more intriguing. This game basically works on human unpredictability and "stabbyness" in that sense. I've relied on the secret draw setting to trick many people. It's a mechanic, a part of the game. And regarding phase results, I think there should be a tally at the end (already suggested) for each player, not just "[country] has lost centres, he has to disband [number] units." 

More like this (for the tally):
Germany: Lost 2 centres, 5 remaining
Italy: Lost 3 centres, 0 remaining

Please consider my suggestion and comment. :)

**Comment by zond** [comment]
RIght. Please create another ticket for the SC tally feature, so we can track them separately :)

**Comment by zond** [comment]
Oh wait, it already has it's own issue: #32

**Comment by Flieh** [comment]
I agree about the need for transparency as far as identity is concerned. I'm not at all advocating for anonymous games (you can keep playing Droidippy if you want that). But I still think the draw should be made by secret ballot. It is a game where we are trying to trick and deceive each other. I think the secret ballot is akin your allies and adversaries not knowing what your orders are going to be. The other signals, nmr, ready and active should absolutely be public. Of course, it's your puppy, so you get to decide and we can certainly adapt to playing open ballot draws. It's not a deal breaker.

**Comment by Flieh** [comment]
And I still think the button should be called DRAW because that is the decision you're being asked to make. DIAS describes the rules but not the decision.
